uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/M2ServletContextListenerTest.java

```diff
@@ -88,6 +97,32 @@ public void deployJarsFrowWar() throws Exception {
         assertThat(pom.exists()).isTrue();
     }
 
+    @Test
+    public void checkRegularExecution() {
+        System.setProperty("kie.maven.offline.force", "false");
+        final M2ServletContextListener listener = spy(new M2ServletContextListener());
+        final ServletContextEvent contextEvent = mock(ServletContextEvent.class);
+        final ServletContext servletContext = mock(ServletContext.class);
+        when(servletContext.getRealPath(any())).thenReturn(".");
+        when(contextEvent.getServletContext()).thenReturn(servletContext);
+        listener.contextInitialized(contextEvent);
+        System.clearProperty("kie.maven.offline.force");
```

Comment:
Please move this `clearProperty` call to `@After` to guarantee cleanup even if the test fails.